Implement the retriving of userdata for an existing user

cworf91 opened the pull request:
please don´t merge until a integration test is done

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/servicecatalog/oscm/376)
<!-- Reviewable:end -->

adamczykba committed:
Implemented token validation

adamczykba committed:
Implemented token validation

adamczykba committed:
Merge branch 'sso-token-validation' of https://github.com/servicecatalog/oscm into sso-token-validation

adamczykba committed:
Merge branch 'master' of https://github.com/servicecatalog/oscm into sso-token-validation

adamczykba committed:
Implemented token validation

adamczykba committed:
Implemented token validation

adamczykba committed:
Disabled libraries directory caching

adamczykba committed:
Merge branch 'sso' of https://github.com/servicecatalog/oscm into sso-token-validation

adamczykba committed:
Applied PR changes

GoebelL committed:
Export setting template.

GoebelL committed:
Export setting template.

GoebelL committed:
Merge branch 'task/removeTenantSettings' of https://github.com/servicecatalog/oscm.git into task/removeTenantSettings

Commit:
Implement the retriving of userdata for an existing user

Commit:
Implement the retriving of userdata for an existing user

Commit:
Implement the retriving of userdata for an existing user

Commit:
Merge branch 'task/sso-retrive_user_Data' of https://github.com/servicecatalog/oscm.git into task/sso-retrive_user_Data

Commit:
new travis run

Commit:
new travis run

adamczykba committed:
Merge pull request #373 from servicecatalog/sso-token-validation

Sso token validation

Crystalzord reviewed (commented):
> Reviewed 5 of 5 files at r1.
> *[Reviewable](https://reviewable.io/reviews/servicecatalog/oscm/376)* status: all files reviewed, 3 unresolved discussions (waiting on @cworf91)

___
*[oscm-identitymgmt/javasrc/org/oscm/identityservice/bean/IdentityServiceBean.java, line 2960 at r1](https://reviewable.io/reviews/servicecatalog/oscm/376#-LkhQRw1D21z3snX2_CT:-LkhQRw1D21z3snX2_CU:b-ou2fzq) ([raw file](https://github.com/servicecatalog/oscm/blob/2431a969cd98ec59ee657de90269f9ee9761b715/oscm-identitymgmt/javasrc/org/oscm/identityservice/bean/IdentityServiceBean.java#L2960)):*
> ```Java
>     }
>     
> ```

Unnecessary empty new lines and whitespaces
___
*[oscm-identitymgmt/javasrc/org/oscm/identityservice/rest/Userinfo.java, line 32 at r1](https://reviewable.io/reviews/servicecatalog/oscm/376#-LkhQIVC9oklYmXnBE0p:-LkhQIVC9oklYmXnBE0q:b-ou2fzq) ([raw file](https://github.com/servicecatalog/oscm/blob/2431a969cd98ec59ee657de90269f9ee9761b715/oscm-identitymgmt/javasrc/org/oscm/identityservice/rest/Userinfo.java#L32)):*
> ```Java
>             .getLogger(Userinfo.class);
> ```

Unnecessary empty new lines and whitespaces
___
*[oscm-identitymgmt/javasrc/org/oscm/identityservice/rest/Userinfo.java, line 97 at r1](https://reviewable.io/reviews/servicecatalog/oscm/376#-LkhQBI-0HQsnUQ4ZLST:-LkhQBI-0HQsnUQ4ZLSU:bp1x4j6) ([raw file](https://github.com/servicecatalog/oscm/blob/2431a969cd98ec59ee657de90269f9ee9761b715/oscm-identitymgmt/javasrc/org/oscm/identityservice/rest/Userinfo.java#L97)):*
> ```Java
>                 return Salutation.MS;
>             default:
>                 logger.logDebug("No gender defined or the gender " + gender + " is unkonwn");
> ```

there is a minor  typo in the log debug- unkonwn instead of unknown


<!-- Sent from Reviewable.io -->

GoebelL committed:
Failed property.
Fix bean properties resolver issue.
Add default tenant.
Export tenant properties.

GoebelL committed:
Failed property.

GoebelL reviewed (changes requested):
> *[Reviewable](https://reviewable.io/reviews/servicecatalog/oscm/376)* status: all files reviewed, 4 unresolved discussions (waiting on @cworf91)

___
*[oscm-identitymgmt/javasrc/org/oscm/identityservice/rest/Userinfo.java, line 128 at r1](https://reviewable.io/reviews/servicecatalog/oscm/376#-Lkhq43j0oxstIvDy2lM:-Lkhq43j0oxstIvDy2lN:bln6aj1) ([raw file](https://github.com/servicecatalog/oscm/blob/2431a969cd98ec59ee657de90269f9ee9761b715/oscm-identitymgmt/javasrc/org/oscm/identityservice/rest/Userinfo.java#L128)):*
> ```Java
>     protected String createUrl(String userId, String tenantId, String token, String host) {
>         StringBuilder url = new StringBuilder();
>         url.append(host); 
> ```

Is the protocol missing here?


<!-- Sent from Reviewable.io -->

GoebelL committed:
Fix bean properties resolver issue.

GoebelL committed:
Add default tenant.

GoebelL committed:
Export tenant properties.

GoebelL committed:
Update button labels.

GoebelL committed:
Update button labels.

GoebelL committed:
Merge branch 'task/removeTenantSettings' of https://github.com/servicecatalog/oscm.git into task/removeTenantSettings

GoebelL committed:
Fixed button label and selection.

GoebelL committed:
Merge pull request #381 from servicecatalog/task/removeTenantSettings

Provide for downloading tenant configuration templates

GoebelL committed:
Added tenant id to login redirect for oscm-identity.

GoebelL committed:
Just formatting.

GoebelL committed:
Changed setup for doFilter_noMPLLogin.

GoebelL committed:
Include tenantID handling.

GoebelL committed:
Added.

GoebelL committed:
Use tenantId parameter in camel case with oscm-identity

kowalczyka committed:
Adapted validation to use tenant information

GoebelL committed:
Merge pull request #384 from servicecatalog/task/removeTenantSettings

Enhanced OIDCFilter to include tenantId handling

GoebelL committed:
Fetch interfaces libs from sso branch.

GoebelL committed:
Merge pull request #388 from servicecatalog/task/libs_from_sso_branch

Take oscm interfaces from SSO branch

Commit:
remove unnecessary special character from travis.yml

GoebelL committed:
Changed SAML_SP to OIDC authentication mode.

GoebelL committed:
Adapt SAML_SP to OIDC mode.

GoebelL committed:
Adapt unit tests.

GoebelL committed:
Merge branch 'sso' into task/oidcAuthMode

GoebelL committed:
Merge pull request #390 from servicecatalog/task/oidcAuthMode

Change to OIDC authentication mode.

GoebelL committed:
Adapt properties import.

GoebelL committed:
Merge branch 'sso' into task/oidcAuthMode

GoebelL committed:
Merge pull request #391 from servicecatalog/task/oidcAuthMode

Adapt properties import

GoebelL committed:
Take interfaces from sso branch.

GoebelL committed:
Merge branch 'task/oidcAuthMode' of https://github.com/servicecatalog/oscm.git into task/oidcAuthMode

GoebelL committed:
Merge branch 'sso' into task/oidcAuthMode

GoebelL committed:
Merge pull request #392 from servicecatalog/task/oidcAuthMode

Take external interfaces from interfaces sso branch

kowalczyka committed:
Access/refresh token handling (#396)

* Add tokens storing in session context

* Resolved unit test issue

adamczykba committed:
Changed protocol for validate and login requests to HTTPS (#393)

* Changed protocol for validate and login requests to HTTPS

* Changed protocol for internal token validation back to HTTP

* Changed protocol for internal token validation back to HTTP

* Merge branch 'sso-ssl' of https://github.com/servicecatalog/oscm into sso-ssl

* Changed hostname for token validation endpoint to contener name

* Fixed typo in url

* Added filter flow break after user error

* Updated token handle to use different claim

* Fixed failing test

* Merge branch 'sso' into sso-ssl

* Merge branch 'sso' into sso-ssl

GoebelL committed:
Fix logout url.

GoebelL committed:
Removed blank lines.

GoebelL committed:
Combined buildLogoutUrl methods.

GoebelL committed:
Merge pull request #397 from servicecatalog/task/fixLogoutURL

Fixed logout URL

GoebelL committed:
Fix description field enabling.

GoebelL committed:
Merge branch 'sso' into task/fixSelection

GoebelL committed:
Merge pull request #398 from servicecatalog/task/fixSelection

Fix description field enabling.

Commit:
Merge branch 'task/sso-retrive_user_Data' of https://github.com/servicecatalog/oscm.git into task/sso-retrive_user_Data

Commit:
remove unused line

Commit:
change url

Commit:
add changes

cworf91 committed:
Just a naming test

cworf91 committed:
Add changes for barer token

cworf91 committed:
fix bug

cworf91 reviewed (commented):
> *[Reviewable](https://reviewable.io/reviews/servicecatalog/oscm/376)* status: 2 of 42 files reviewed, 4 unresolved discussions (waiting on @Crystalzord and @GoebelL)

___
*[oscm-identitymgmt/javasrc/org/oscm/identityservice/bean/IdentityServiceBean.java, line 2960 at r1](https://reviewable.io/reviews/servicecatalog/oscm/376#-LkhQRw1D21z3snX2_CT:-LmcwnkEBLXmpofm75AI:b-896fix) ([raw file](https://github.com/servicecatalog/oscm/blob/2431a969cd98ec59ee657de90269f9ee9761b715/oscm-identitymgmt/javasrc/org/oscm/identityservice/bean/IdentityServiceBean.java#L2960)):*
<details><summary><i>Previously, Crystalzord (Lukasz Wiecek) wrote…</i></summary><blockquote>

Unnecessary empty new lines and whitespaces
</blockquote></details>

Done.
___
*[oscm-identitymgmt/javasrc/org/oscm/identityservice/rest/Userinfo.java, line 32 at r1](https://reviewable.io/reviews/servicecatalog/oscm/376#-LkhQIVC9oklYmXnBE0p:-LmcwoUdBeBTPDx0_6Z5:b-896fix) ([raw file](https://github.com/servicecatalog/oscm/blob/2431a969cd98ec59ee657de90269f9ee9761b715/oscm-identitymgmt/javasrc/org/oscm/identityservice/rest/Userinfo.java#L32)):*
<details><summary><i>Previously, Crystalzord (Lukasz Wiecek) wrote…</i></summary><blockquote>

Unnecessary empty new lines and whitespaces
</blockquote></details>

Done.
___
*[oscm-identitymgmt/javasrc/org/oscm/identityservice/rest/Userinfo.java, line 97 at r1](https://reviewable.io/reviews/servicecatalog/oscm/376#-LkhQBI-0HQsnUQ4ZLST:-Lmcwp6uEjIX01D7cTgJ:b-896fix) ([raw file](https://github.com/servicecatalog/oscm/blob/2431a969cd98ec59ee657de90269f9ee9761b715/oscm-identitymgmt/javasrc/org/oscm/identityservice/rest/Userinfo.java#L97)):*
<details><summary><i>Previously, Crystalzord (Lukasz Wiecek) wrote…</i></summary><blockquote>

there is a minor  typo in the log debug- unkonwn instead of unknown
</blockquote></details>

Done.
___
*[oscm-identitymgmt/javasrc/org/oscm/identityservice/rest/Userinfo.java, line 128 at r1](https://reviewable.io/reviews/servicecatalog/oscm/376#-Lkhq43j0oxstIvDy2lM:-LmcwsJa8px4DbCMvqlg:b-896fix) ([raw file](https://github.com/servicecatalog/oscm/blob/2431a969cd98ec59ee657de90269f9ee9761b715/oscm-identitymgmt/javasrc/org/oscm/identityservice/rest/Userinfo.java#L128)):*
<details><summary><i>Previously, GoebelL (Lorenz Goebel) wrote…</i></summary><blockquote>

Is the protocol missing here?
</blockquote></details>

Done.


<!-- Sent from Reviewable.io -->

cworf91 committed:
refactor
